galsim/phase_screens.py

```diff
@@ -0,0 +1,646 @@
+    r0 ~ wavelength^(6/5).  To reduce confusion, the input parameter here is named r0_500 and refers
+    explicitly to the Fried parameter at a wavelength of 500 nm.  The outer scale is typically in
+    the 10s of meters and does not vary with wavelength.
+
+    To create multiple layers, simply specify keyword arguments as length-N lists instead of scalars
+    (works for all arguments except `time_step` and `rng`).  If, for any of these keyword arguments,
+    you want to use the same value for each layer, then you can just specify the argument as a
+    scalar and the function will automatically broadcast it into a list with length equal to the
+    longest found keyword argument list.  Note that it is an error to specify keywords with lists of
+    different lengths (unless only one of them has length > 1).
+
+    The one exception to the above is the keyword `r0_500`.  The effective Fried parameter for a set
+    of atmospheric layers is r0_500_effective = (sum(r**(-5./3) for r in r0_500s))**(-3./5).
+    Providing `r0_500` as a scalar or length-1 list will result in broadcasting such that the
+    effective Fried parameter for the whole set of layers equals the input argument.
+
+    As an example, the following code approximately creates the atmosphere used by Jee+Tyson(2011)
+    for their study of atmospheric PSFs for LSST.  Note this code takes about ~3 minutes to run on
+    a fast laptop, and will consume about (8192**2 pixels) * (8 bytes) * (6 screens) ~ 3 GB of
+    RAM in its final state, and more at intermediate states.
+
+        >>> altitude = [0, 2.58, 5.16, 7.73, 12.89, 15.46]  # km
+        >>> r0_500_effective = 0.16  # m
+        >>> weights = [0.652, 0.172, 0.055, 0.025, 0.074, 0.022]
+        >>> r0_500 = [r0_500_effective * w**(-3./5) for w in weights]
+        >>> speed = np.random.uniform(0, 20, size=6)  # m/s
+        >>> direction = [np.random.uniform(0, 360)*galsim.degrees for i in xrange(6)]
+        >>> npix = 8192
+        >>> screen_scale = r0_500_effective
+        >>> atm = galsim.Atmosphere(r0_500=r0_500, screen_size=screen_scale*npix, time_step=0.005,
+                                    altitude=altitude, L0=25.0, speed=speed,
+                                    direction=direction, screen_scale=screen_scale)
+
+    Once the atmosphere is constructed, a 15-sec exposure length monochromatic PSF at 700nm (using
+    an 8.4 meter aperture, 0.6 fractional obscuration and otherwise default settings) takes about
+    7 minutes to generate on a fast laptop.
+
+        >>> psf = atm.makePSF(lam=700.0, exptime=15.0, diam=8.4, obscuration=0.6)
+
+    Many factors will affect the timing of results, of course, including aperture diameter, gsparams
+    settings, pad_factor and oversampling options to makePSF, time_step and exposure time, frozen
+    vs. non-frozen atmospheric layers, and so on.  We recommend that users try varying these
+    settings to find a balance of speed and accuracy.
```

Comment:
Accuracy seems like the wrong word here.  It's not like there would be some truth value to compare to.  Maybe "fidelity" captures what you mean better?  (Or maybe not.  It's kind of a synonym, but maybe with a slightly different connotation.)